### ISSUE TITLE
[YAML branch] No Feeds Found - data_pp.py changes pair standards

### ISSUE THREAD

**Issue by trentmc** [oceanprotocol/pdr-backend#416]
### Background / motivation

(From Roberto, [here](https://github.com/oceanprotocol/pdr-backend/issues/411#issuecomment-1830951893).)

Hi @trentmc while running through vps.md Barge I am noticing the following
(1) Important - Bots are breaking because they can't find a feed. (changes in pair_str from "/" to "-")
(2) Minor - `barge/pdr-publisher` deploys `BTC/TUSD`

I've been merging your latest code from yaml-cli2 over to this branch (issue411) such that I'm running off your codebase vs. VPS deployment.

**[1. No Feeds Found - data_pp.py changes pair standards]**
Problems between `data_pp.py` and `subgraph.py` such that contracts ["ntfData"]["pair"]  doesn't match pair_str.

**Problem:**
1. data_pp.pair_strs now returns ['ETH-USDT'], but web3_pp.get_feeds() expects ['ETH/USDT']
2. I believe we should maintain all pair definitions such as to only use "/", and avoid using "-" everywhere. 1 or the other, not both. I'm not sure what the motivation was, but at the contract level and many other places (i,.e, APIs) this breaks convention. I don't believe this should be adopted.
3. I'm not sure how this isn't failing the tests ATM... I'm gonna run testing flow for barge vps, so i can see what's going on with coverage

**TLDR; In base_predictoor_agent.py ln 41**
```
  # TODO: data_pp.pair_strs returns ['ETH-USDT'], but web3_pp.get_feeds() expects ['ETH/USDT']
  x = list(self.ppss.data_pp.pair_strs)
  y = list([self.ppss.data_pp.timeframe])
  z = list(self.ppss.data_pp.exchange_strs)

  cand_feeds = self.ppss.web3_pp.get_feeds(x, y, z)
```

subgraph.py - ln 217, 218
```
info725 = contract["token"]["nft"]["nftData"]
```
![image](https://github.com/oceanprotocol/pdr-backend/assets/69865342/6d3fbec3-231c-47a0-a6d8-132eef842a84)

### Background 2: How did unit tests miss this?

(From Roberto, [here](https://github.com/oceanprotocol/pdr-backend/issues/411#issuecomment-1831011787).)

Answer as to why this isn't covered in tests.
1. predictoor agent mocks the query, and doesn't enforce any pair_str format from the contract
```
@enforce_types
def run_agent_test(tmpdir, monkeypatch, predictoor_agent_class):
    monkeypatch.setenv("PRIVATE_KEY", PRIV_KEY)

    ppss = _ppss(tmpdir)

    mock_query = MockQuery(ppss.data_pp)
    monkeypatch.setattr(
        "pdr_backend.ppss.web3_pp.query_feed_contracts",
        mock_query.mock_query_feed_contracts,
    )
```

2. test_web3_pp.py assumes that contract returns `pair: BTC-ETH` but that is not correct
```
FEED_DICT = {  # info inside a predictoor contract
    "name": "Contract Name",
    "address": ADDR,
    "symbol": "test",
    "seconds_per_epoch": 300,
    "seconds_per_subscription": 60,
    "trueval_submit_timeout": 15,
    "owner": "0xowner",
    "pair": "BTC-ETH", # This is not correct
    "timeframe": "1h",
    "source": "binance",
}

**Comment by trentmc** [comment]
Fixed in yaml-cli2 branch in commit https://github.com/oceanprotocol/pdr-backend/commit/06aa946e02cec953b21d4842b6b098a07c118580
